### ISSUE TITLE
Generate address book automatically

### ISSUE THREAD

**Issue by marcelstanley** [cartesi/rollups-node#233]
As hinted at https://github.com/cartesi/rollups-node/pull/204#discussion_r1443286056_, the address book used for testing is hardcoded but, ideally, should be generated automatically.

Let's do this via `go generate`.

**Comment by marcelstanley** [comment]
This is specially important after emulator SDK updates, where the application address changes.

This also affects devnet generation.

**Comment by tuler** [comment]
Devnet in the CLI uses a zero templateHash and zero salt. So it’s not affected by emulator changes. We didn’t see value in dynamically generation in the CLI other than adding complexity to something which can be simple.

**Comment by marcelstanley** [comment]
Some comments/suggestions spread across the repo to be considered:
- https://github.com/cartesi/rollups-node/pull/512#discussion_r1674664475

**Comment by marcelstanley** [comment]
> Devnet in the CLI uses a zero templateHash and zero salt. So it’s not affected by emulator changes. We didn’t see value in dynamically generation in the CLI other than adding complexity to something which can be simple.

Yes but, contrary to our expectations, the address of the application used in devnet is always changed after emulator SDK changes:
- https://github.com/cartesi/rollups-node/commit/2dba529ec4bd08573d211b4bc9ebb9fa0b31ba88
- https://github.com/cartesi/rollups-node/pull/394

Anyway, I just adjusted the scope of this ticket to also consolidate all address configuration in one place

**Comment by vfusco** [comment]
this is already implemented and improved
